toolkit/tools/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.4.0
 	github.com/ulikunitz/xz v0.5.7
-	golang.org/x/sys v0.0.0-20200509044756-6aff5f38e54f
+	golang.org/x/sys v0.0.0-20201207223542-d4d67f95c62d
```

Comment:
If this is being versioned bumped please also update the cgmanifest.

`toolkit/tools/cgmanifest.json`